Lib/gftools/builder/dependencies.py

```diff
@@ -0,0 +1,163 @@
+"""
+Similar to pip freeze, but only listing the installed dependencies for the selected package.
+
+E.g.:
+    $ python freeze_requirements.py fonttools[ufo]
+    # Installed requirements for 'fonttools[ufo]' (5 in total):
+
+    appdirs==1.4.4
+    fonttools==4.24.4
+    fs==2.4.13
+    pytz==2021.1
+    six==1.16.0
+
+Credit to Cosimo Lupo (anthrotype) for starting this as a gist:
+https://gist.github.com/anthrotype/531a425c8a0ba5ee975bc2ec8add7b82
+"""
+
+__requires__ = ["packaging", "importlib_metadata; python_version < '3.8'"]
+
+from collections import deque, defaultdict
+import re
+from typing import (
+    Generator,
+    Iterable,
+    DefaultDict,
+    Dict,
+    Set,
+    Tuple,
+)
+from fontTools.ttLib import TTFont, newTable
+
+try:
+    import importlib.metadata as importlib_metadata
+except ImportError:
+    import importlib_metadata  # type: ignore
```

Comment:
I think you no longer need this importlib_metadata backport since you should be already only supporting python_requires=">=3.8" in setup.py, if I am not mistaken